datavirt64/dynamicvdb-datafederation/app/datagrid-materialization/pom.xml

```diff
@@ -0,0 +1,148 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.jboss.teiid.quickstart</groupId>
+    <artifactId>jdg-remote-cache-materialization</artifactId>
+    <version>3.0.0</version>
+    <name>JDG Remote Cache Materialization Quickstart</name>
+    <description>This quickstart demonstrates how to use a remote JDG cache for materialization</description>
+
+    <properties>
+
+        <!-- Where to put the EAP module. -->
+        <module.output.dir>${basedir}/../modules/com/client/quickstart/addressbook/pojos/main</module.output.dir>
+
+        <!-- Defines the main class to be execute when running java -jar command -->
+        <main.class.remote-query>org.jboss.as.quickstarts.datagrid.hotrod.query.AddressBookManager</main.class.remote-query>
+
+        <version.org.infinispan.6>6.4.0.Final-redhat-4</version.org.infinispan.6>  
```

Comment:
should be using a JDG 7.1.1 version instead (which would be 8.4.something I guess)